app/src/main/res/layout/view_bottom_spacer.xml

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think we could drop the "view_" suffix if you wanted to. 

Can you also add the mozilla license and a new line at the EOF to appease the lint?

Reply:
done and done